microsoft/testsuites/kselftest/kselftest-suite.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict
+
+from lisa import (
+    Environment,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    schema,
+    search_space,
+    simple_requirement,
+)
+
+from lisa.testsuite import TestResult
+from lisa.util import MissingPackagesException, SkippedException
+from microsoft.testsuites.kselftest.kselftest import Kselftest
+
+@TestSuiteMetadata(
+    area="kselftest",
+    category="community",
+    description="""
+    This test suite is used to run kselftests.
+    """,
+)
+class KselftestTestsuite(TestSuite):
+    _TIME_OUT = 12000
+
+    @TestCaseMetadata(
+        description="""
+        This test case will run kselftests tests.
+        """,
+        priority=3,
```

Comment:
Can you please ensure that the priority is correct: https://mslisa.readthedocs.io/en/main/run_test/microsoft_tests.html#test-priority?

Reply:
I have used the same priority as ltp. Should this be set to 4 instead?

Reply:
It's upto your use case. My comment was to ensure that it is set correctly